src/attr.rs

```diff
@@ -69,8 +69,8 @@ impl<'string> AttrValue<'string> {
 ///
 /// Note that this implementation does not differentiate between [`AttrValue::String`] and
 /// [`AttrValue::Bytes`].
-impl PartialEq for AttrValue<'_> {
-    fn eq(&self, other: &AttrValue<'_>) -> bool {
+impl<'string> PartialEq for AttrValue<'string> {
+    fn eq(&self, other: &AttrValue<'string>) -> bool {
```

Comment:
This is likely not the right fix -- instead, `Self::Bytes` below on line 79 should be changed to AttrValue::Bytes to avoid requiring the lifetimes to be related.

Reply:
Thanks. Updated patch to use your suggested modification instead. (Had to do the same on a few lines)